workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/Model/Details/DetailsStore.cs

```diff
@@ -297,41 +298,25 @@ private static List<string> ExtractWorkerTypes(string path)
         {
             Logger.Trace($"Extracting worker types from {path}.");
 
-            const string workerTypeFlag = "+workerType";
             var workerTypes = new List<string>();
 
-            var fileNames = Directory.EnumerateFiles(path, "*.json").ToList();
+            var fileNames = Directory.EnumerateFiles(path, "spatialos.*.worker.json").ToList();
             Logger.Trace($"Found {fileNames.Count} worker json files:\n - {string.Join("\n - ", fileNames)}");
 
             foreach (var fileName in fileNames)
             {
                 Logger.Trace($"Extracting worker type from {fileName}.");
-                var text = File.ReadAllText(fileName);
-                if (!text.Contains(workerTypeFlag))
-                {
-                    Logger.Warn($"{fileName} does not contain the '{workerTypeFlag}' flag.");
-                    continue;
-                }
+                var match = Regex.Match(fileName, @"spatialos\.(.*)\.worker\.json");
 
-                var jsonRep = JObject.Parse(text);
-                var arguments = jsonRep.SelectToken("external.default.windows.arguments");
-                if (arguments == null)
+                if (!match.Success)
```

Comment:
Might be worth logging a warning here if we match a worker json but can't get a worker type out of it